app/grandchallenge/reader_studies/forms.py

```diff
@@ -161,8 +161,9 @@ class Meta(ReaderStudyCreateForm.Meta):
             "hanging_list": (
                 "A list of hangings. "
                 "The hanging defines which image (the hanging value) "
-                "should be assigned to which image port or overlay.  "
-                'e.g., [{"main":"im1.mhd","secondary":"im2.mhd", "secondary-overlay": "im2_overlay.mhd"}]'
+                "should be assigned to which image port or overlay. Options are: "
+                "main, secondary, tertiary, quaternary, quinary, senary, septenary, octonary, nonary, denary. "
+                'For instance: [{"main":"im1.mhd","main":"im1-overlay.mhd"}]'
```

Comment:
`[{"main":"im1.mhd","main":"im1-overlay.mhd"}]` would not be a valid hanging protocol. Also check you line length, limit is 80 chars (not enforced right now, but will be).